apps/omg_watcher_info/test/omg_watcher_info/api/block_test.exs

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.API.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.Fixtures
+  use OMG.WatcherInfo.Fixtures
+  use OMG.Watcher.Fixtures
+
+  alias OMG.WatcherInfo.DB
+  alias OMG.WatcherInfo.API.Block
+
+  describe "get_block/1" do
+    @tag fixtures: [:initial_blocks]
+    test "returns block by id" do
+      block_hash = "##{1000}"
```

Comment:
Can we just `block_hash = "#1000"`? The `#{...}` is used for accessing a variable.

Same applies to another test.

Reply:
Done with ce51218.